at_client/lib/src/service/encryption_service.dart

```diff
@@ -478,4 +481,50 @@ class EncryptionService {
         EncryptionUtil.decryptBytes(fileContent, decryptionKey);
     return encryptedValue;
   }
+
+  Future<File> encryptFileInChunks(
+      File inputFile, String fileEncryptionKey, int chunkSize) async {
+    var chunkedStream = ChunkedStreamReader(inputFile.openRead());
+    final length = inputFile.lengthSync();
+    var readBytes = 0;
+    final fileName = inputFile.uri.pathSegments.last;
+    final encryptedFile = File('${inputFile.parent.path}/encrypted_$fileName');
```

Comment:
'/' needs to be replaced with Platform.pathSeparator.

Reply:
done